include/net/gptp.h

```diff
@@ -27,11 +27,10 @@
 extern "C" {
 #endif
 
-#define GPTP_CLOCK_ACCURACY_UNKNOWN        0xFE
-
-#define GPTP_OFFSET_SCALED_LOG_VAR_UNKNOWN 0x436A
+#define GPTP_OFFSET_SCALED_LOG_VAR_UNKNOWN 0x6A43
```

Comment:
Perhaps comment here explaining what this value is and in what endianess.

Reply:
That is actually a very good question. The value was switched based on observations of the packets that openAvnu sends (as it made sense to mimic it in the beginning as it is something that is _known to work_) - unfortunatelly I cannot find any references to that value in 802.1AS - I will test it all again without this change, and if it still works I guess we can temporarily drop this patch until the question is properly answered.

Reply:
Perhaps trying to find out the value handling from OpenAvnu sources could help here. There is lot of C++ code there and it is not easy to spot some specific place, I have tried something like this earlier.

Reply:
Unfortunately the value is hard-coded there too (and without any comments):
https://github.com/AVnu/gptp/blob/98e3adaeb4c5a9534d24aee166d3690ea6a5c7f4/common/ieee1588clock.cpp#L95

The definition is the same as it was here before this patch, but Zephyr does `htons` on this value before sending this packet, openAvnu sends it as it is, so we needed to switch it to make it the same.

I will let you know how the tests without this patch go, I will also try to go through the standards to see if there are some details and suggested default values for this field.

Reply:
Ok, that is tricky then. I wonder if linux-ptp daemon has anything better documented in its code.

Reply:
I removed the patch for now as this value doesn't seem to affect sync process.

Reply:
But for the sake of completeness:
* linux-ptp sets this field to `0xffff` by default
* 802.1as-2001, 8.6.2.4b says, that if `offsetScaledLogVariance` is not specified or not know, then this field should be set to `0x4100` - maybe that is the way to go? None of the daemons we used use this value, but it seems to be the best choice actually (although since it doesn't seem to have any effect on sync then it is kind of low priority)